v1/spec/lib/v1/field_spec.rb

```diff
@@ -6,43 +6,41 @@ module V1
     let(:resource) { 'item' }
     let(:item_mapping) {
       {
-        'id' => { 'type' => 'string', 'index' => 'not_analyzed', 'sort' => 'field' },
-        'title' => { 'type' => 'string' },
-        'description' => { 'type' => 'string' },
+        'id' => { 'type' => 'keyword', 'sort' => 'field' },
+        'title' => { 'type' => 'text' },
+        'description' => { 'type' => 'text' },
```

Comment:
These new `text` and `keyword` types are nice - looks like the schema will be easier to maintain now.